examples/suit_update/README.md

```diff
@@ -483,10 +484,281 @@ success, the application reboots on the new slot:
     Running from slot 1
     ...
 
-
 The slot number should have changed from after the application reboots.
 You can do the publish-notify sequence several times to verify this.
 
+## Testing Without Hardware
```

Comment:
How about we split the Readme? It starting to be huge and difficult to read.

Reply:
I agree, I'll create a separate `README.native.md` with the new content

Reply:
+1